log-insights/setup/opentelemetry.mdx

```diff
@@ -31,18 +78,122 @@ In values.yaml file, update the section of `service`:
+{`service:
+  create: true
+  name: pganalyze-collector-otel-service
+  type: ClusterIP
+  port: 4318
+  targetPort: 4318
+  ports:
+    - name: pganalyze-collector-otel1
+      port: 4318
+      targetPort: 4318
+    - name: pganalyze-collector-otel2
+      port: 4319
+      targetPort: 4319`}
+</CodeBlock>
+
+When there is a `ports` section, the Helm chart will ignore the settings 
+for any indidivual `port` and `targetPort` settings.
+
+
+## Fluent Bit configuration
+If you are using [Fluent Bit](https://fluentbit.io/) as a telemetry agent, you need to specify
+the correct INPUT and OUTPUT plugin settings to send logs to the pganalyze collector's OpenTelemetry 
+HTTP endpoint. Assuming that the Fluent Bit pod is correctly configured as a DeamonSet, the 
+following INPUT and OUTPUT configuration examples should successfully start sending logs from
+each Postgres pod to the pganalyze collector for processing. You will need to adjust your input
+path accordingly.
+
+<CodeBlock language="text">
+{`[INPUT]
+    name  tail
+    path  /var/log/postgresql/cluster-example-*_default_postgres*.log
+    Tag kube.*
+    multiline.parser docker, cri
+    DB /var/log/flb_kube.db`}
+</CodeBlock>
 
 <CodeBlock language="text">
-{`  [OUTPUT]
-    name  opentelemetry
-    match kube.*postgres*
-    host  pganalyze-collector-otel-service
-    port  4318`}
+{`[OUTPUT]
+  name  opentelemetry
+  match kube.*postgres*
+  host  pganalyze-collector-otel-service
+  port  4318`}
 </CodeBlock>
 
 With this flow, Fluent Bit tails the logs of Postgres pods, sends the logs to the
 pganalyze collector's OpenTelemetry HTTP endpoint (via `pganalyze-collector-otel-service`),
 and the collector processes the logs.
+
+If you have multiple Postgres clusters, you can set up multiple Fluent Bit OUTPUT
+plugins, each pointing to a different OpenTelemetry HTTP endpoint on the pganalyze collector. 
+
+<CodeBlock language="text">
+{`[OUTPUT]
+    Name  opentelemetry
+    Match kube.*cluster-example2-*
+    Host  192.168.12.242
+    Port  4319
+ 
+[OUTPUT]
+    Name  opentelemetry
+    Match kube.*cluster-example-*
+    Host  192.168.12.242
+    Port  4318`}
+</CodeBlock>
+
+### Troubleshooting Fluent Bit and other Protobuf payloads
+One of the common issues when using Fluent Bit with OpenTelemetry is that
+the payloads are sent in Protobuf format which is a compact, efficient transport
+format, yet difficult to debug on the receiving end when it doesn't match the
+expected OpenTelemetry log data model.
+
+To assist with troubleshooting, you can set the `--very-verbose` runtime parameter
+for the pganalyze collector. This will log the raw Protobuf payloads received
+by the OpenTelemetry HTTP server, which can help you identify if the payloads
+are correctly formatted and match the expected OpenTelemetry log data model.
+
+For example, assuming that `pganalyze-collector.conf` file is in the expected `/etc` location,
+you can run the collector with debug logs enabled like this:
```

Comment:
I've rarely (ever?) seen a pganalyze-collector.conf in a different location, but people may be running the collector as a container. So maybe reword like this:

```suggestion
For example, with a package-based deployment of the collector,
you can run the collector with debug logs enabled like this:
```